docs/components/menus/index.html

```diff
@@ -13,125 +14,100 @@ <h2 class="hxSectionTitle" id="menu">Basic Menu</h2>
       <hx-disclosure class="hxBtn" aria-controls="menuId">
         <hx-icon type="cog"></hx-icon>
         Actions
-        <hx-icon class="menu-icon" type="angle-down"></hx-icon>
+        <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
       </hx-disclosure>
-      <hx-menu class="hxBox-sm" id="menuId">
-        <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
-        <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
-        <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+      <hx-menu id="menuId">
+        <hx-menuitem>Action 1</hx-menuitem>
+        <hx-menuitem>Action 2</hx-menuitem>
+        <hx-menuitem>Action 3</hx-menuitem>
       </hx-menu>
     </div>
     {% code 'html' %}
```

Comment:
Should these code blocks on this page also have `{% raw %}` blocks around them like in the other component pages above? Not sure if that's what Steve meant.

Reply:
question applies to rest of PR

Reply:
`{% raw %}` is only needed if I need to use `{{ variable }}` syntax for a VueJS demo. Because both nunjucks and VueJS use the same variable syntax, I have to use `{% raw %}` to keep nunjucks from trying to interpolate a nonexistent variable.